inst/include/Rcpp/Reference.h

```diff
@@ -59,15 +59,15 @@ namespace Rcpp{
         Reference_Impl( const std::string& klass ) {
             SEXP newSym = Rf_install("new");
             Shield<SEXP> call( Rf_lang2( newSym, Rf_mkString( klass.c_str() ) ) );
-            Storage::set__( Rcpp_eval( call ) ) ;
+            Storage::set__( Rcpp_eval( call , Rcpp::internal::get_Rcpp_namespace()) );
```

Comment:
Ditto here.   If you can think of better alternatives I'd be all ears :)   I guess worked Kurt worked this out "the hard way" as recent changes in R-devel required it.

In his email to me, he stressed the use of `R_DEFAULT_PACKES=NULL` with `debug(Rcpp::loadModule)` which lead RcppBDT loading to fail -- via either  `loadNamespace()` or `library()` leading to this `Module( module, mustStart = TRUE, where = env )` failing for
lack of `new`.

Personally, I find this whole dance around the methods package mysterious. If it were me, I'd just depend on it...